index.html

```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
   <a href="#">See projects</a>
 </p>
 
-</div>
+</section >
 
  <div class="card-works">
+  
+    <p class="abt-self"> About myself</p>
```

Comment:
- [ ] kindly consider adding this in hx tag to meet the below requirement 
> The title (e.g., "About myself") uses a header tag hx
it might be h2,h3,...... not <header> tag

Reply:
kindly be clear. "it might be h2,h3,...... not" what  is this  supposed to mean . not what ?